examples/flutter_gallery/lib/gallery/item.dart

```diff
@@ -158,6 +158,13 @@ List<GalleryItem> _buildGalleryItems() {
       routeName: TabsFabDemo.routeName,
       buildRoute: (BuildContext context) => new TabsFabDemo(),
     ),
+    new GalleryItem(
+      title: 'Floating action button motion',
+      subtitle: 'Action buttons with customized motion',
+      category: 'Material Components',
+      routeName: FabMotionDemo.routeName,
+      buildRoute: (BuildContext context) => new FabMotionDemo(),
+    ),
```

Comment:
might be better to just have this demo replace the existing FAB demo?

Reply:
The two show off different sets of behavior.  

If we want to consolidate demos, then I think it's better to turn this into a Bottom App Bar demo, that way we can show how the two interact during the motion.  That is a change I'd prefer to keep separated from just landing these Floating Action Button changes.